src/services/EdFi.DataManagementService.Api.Tests.E2E/Management/PlaywrightContext.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Playwright;
+
+namespace EdFi.DataManagementService.Api.Tests.E2E.Management;
+
+public class PlaywrightContext
+{
+    private static Task<IAPIRequestContext>? _requestContext;
+
+    private string _apiURL = "http://localhost:5198/";
```

Comment:
Restating a prior question: what is this URL for? Our app is starting up at 8080 by default (I think), so I'm curious what this URL is.

Reply:
Interesting, my solution always start in 5198. I added this as the default URL, unless changed by TestContainers

Reply:
The HTTP tests also start with 5198 by default https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/blob/main/tests/RestClient/test.http

Reply:
You're right, `dotnet run` does use 5198 on my system. Our Docker container uses 8080, which is what I was thinking about. Can change the port when running the container by setting env var `ASPNETCORE_HTTP_PORT`.